docs/install/install-aur.md

```diff
@@ -6,6 +6,37 @@ sidebar_position: 4
 
 # Installing Vignette using AUR
 
-something
+If you wish to install Vignette on your Arch Linux computer, you may use any AUR helper of you choice. The package is [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/). In the example below, I will be using yay.
+
+
+
+## Installation process
+
+In order to install Vignette using yay, please run the following command: 
```

Comment:
I'd say do it like this. It really sounds like you're begging them to run the command... 

How does it sound without?
```suggestion
In order to install Vignette using yay, run the following command:

```

Reply:
fine by me